main/nginx/APKBUILD

```diff
@@ -8,8 +8,8 @@
 #     - CVE-2017-7529
 #
 pkgname=nginx
-pkgver=1.12.2
-pkgrel=6
+pkgver=1.14.0
+pkgrel=0
 # Revision of nginx-tests to use for check().
 _tests_hgrev=cdd44ff602db
```

Comment:
If you change it to the latest commit d6daf03478ad, only 2 tests will fail.

Reply:
There is also the question of if the new njs module should be added as a subpackage - http://nginx.org/en/docs/njs_about.html

Reply:
I think njs needs separate PR